### ISSUE TITLE
FEAT: For Beekeeper-Studio, multiple international languages can be switched.

### ISSUE THREAD

**Issue by PShiYou** [beekeeper-studio/beekeeper-studio#2493]
Personal experience: As a user whose native language is Chinese, when I use Beekeeper-Studio, although I can barely read and understand the English interface, I still feel a certain degree of difficulty.

Global user diversity: As the number of users of Beekeeper-Studio worldwide continues to increase, the language diversity among user groups is also becoming increasingly significant. Users from different countries and regions may use different native languages, which leads to language barriers when they understand and use tools. By adding a multi-language switching function, the experience of these users can be significantly improved and help them use tools more efficiently.

**Comment by not-night-but** [comment]
I agree that we can definitely do better to add more localization to the app, although this would be a big job so we would be looking for contributors to translate the app/implement the localization

**Comment by juratjan123** [comment]
#2912 A PR has been submitted for this issue.

**Comment by PShiYou** [comment]
> [#2912](https://github.com/beekeeper-studio/beekeeper-studio/pull/2912) A PR has been submitted for this issue. [#2912](https://github.com/beekeeper-studio/beekeeper-studio/pull/2912) 已针对此问题提交 PR。

感谢，如果pr成功了，我后面下载来体验一下
